bin/lint

```diff
@@ -1 +1,16 @@
-go list ./... | xargs go run vendor/github.com/golang/lint/golint/*.go
+#!/bin/bash
+
+# Run code Linter for project packages
+#
+# Usage example:
+#> bin/lint
+#> bin/lint ./...
+#> bin/lint ./communication
+#> bin/lint ./communication/...
+
+ARGUMENTS=$@
+if [ -z "$ARGUMENTS" ]; then
+    ARGUMENTS=`go list ./... | grep -v /vendor/`
```

Comment:
grepping here is not necessary - `go list ./...` already excludes vendor packages. And since we're planning to do compilation using docker - we can choose not to use old `Go` version.